ddtrace/contrib/unittest/patch.py

```diff
@@ -230,6 +230,45 @@ def _is_invoked_by_cli(instance: unittest.TextTestRunner) -> bool:
     )
 
 
+def _extract_test_method_object(test_object):
+    test_method_object = None
+    if hasattr(test_object, "_testMethodName"):
+        test_method_object = getattr(test_object, test_object._testMethodName, None)
+    return test_method_object
+
+
+def _get_source_file_path_for_test_method(test_object):
+    test_name = _extract_test_method_name(test_object)
+    test_method_object = _extract_test_method_object(test_object)
+    source_file_path = os.path.relpath(inspect.getfile(test_method_object))
+    if not test_method_object:
+        log.debug(
+            "Tried to collect source file path for test %s but the original test method could not be found",
+            test_name,
+        )
+    return source_file_path
+
+
+def _get_source_lines_for_test_method(test_object):
+    start_line, end_line = None, None
+    test_name = _extract_test_method_name(test_object)
+    test_method_object = _extract_test_method_object(test_object)
+    if not test_method_object:
+        log.debug(
+            "Tried to collect source start/end lines for test %s but the original test method could not be found",
+            test_name,
+        )
+        return start_line, end_line
```

Comment:
I'd explicitly `return None, None` here. I know it's analogous to `start_line, end_line` since they're also both `None` at this point, but I think `return None, None` is a much clearer indication that we're not returning actual data.